packages/graph/hash_graph/bin/hash_graph/src/main.rs

```diff
@@ -168,6 +184,23 @@ async fn stop_gap_setup(pool: &PostgresStorePool<NoTls>) -> Result<(), GraphErro
         }
     }
 
+    // Seed the entity types if they don't already exist
+    let title = link_entity_type.title().to_owned();
+    if connection
+        .create_entity_type(
+            link_entity_type,
+            OwnedById::new(root_account_id),
+            CreatedById::new(root_account_id),
+        )
+        .await
+        .change_context(GraphError)
+        .is_err()
```

Comment:
We improved our error messaging a little a while back. Is it possible to **easily** check if this _is_ an already exists error as opposed to some other kind of error?

I think we want to panic if it's something else like a malformed entity type or something that breaks a constraint, etc.

Reply:
If it's not easy then that's fine, this is a stop_gap_setup

Reply:
Applied in 7292ac09f8eeb81389f73bf65edff16856a85f02